src/betterproto/plugin/models.py

```diff
@@ -139,6 +165,7 @@ def get_comment(
 class ProtoContentBase:
     """Methods common to MessageCompiler, ServiceCompiler and ServiceMethodCompiler."""
 
+    source_file: FileDescriptorProto
```

Comment:
turns out tracking the source_file is necessary to ensure we look in the right file to find comments, in case there are multiple source proto files in scope.